src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -17,6 +17,7 @@
 
 import six
 from six.moves.urllib.request import urlopen  # pylint: disable=import-error
+from azure.cli.core._environment import get_config_dir
```

Comment:
Consider delay this import to the function where it is actually used.

Reply:
Done